alpine-38.dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# apt-utils seems missing and warnings are shown, so we install it.
+RUN apk update && \ 
```

Comment:
Is this equivalent to `apt-get upgrade --yes --force-yes`?

Reply:
Yes, apk (alpine package manager) is written with containers in mind hence it doesnt require these options.